### PR TITLE
Grocery lists

### PR DESCRIPTION
**Pull request by indirap**

### CODE REVIEW

File: imports/api/groceries.js [hunk 1]
```diff
@@ -27,8 +28,9 @@ Meteor.methods({
 		}
 		
 		Groceries.insert({
-			name: name,
-			amount: amount,
+			name,
```
Comment (vatthikorn) [inline imports/api/groceries.js:31]: Didn’t know that not putting the attribute name works

File: server/main.js [hunk 2]
```diff
@@ -2,9 +2,10 @@ import { Meteor } from 'meteor/meteor';
 import { Accounts } from 'meteor/accounts-base';
 
 import '../imports/api/groceries';
+import '../imports/api/grocerylists';
 import '../imports/api/inventories';
 import '../imports/api/recipes';
 
 Meteor.startup(() => {
-	console.log('Meteor started!')
+	console.log('Meteor started!'); // eslint-disable-line
```
Comment (vatthikorn) [inline server/main.js:10]: What does this do? Make eslint ignore it?

**Review by vatthikorn — APPROVED** [review]

File: imports/api/groceries.js (hunk 1 above)
Comment (indirap) [reply imports/api/groceries.js:31]: Yup! This is a feature in ES6 I believe. If you have a variable such as `const someVar = 2;`, then you can do `const newObj = { someVar };` and that's equivalent to `const newObj = { someVar: someVar };`

File: server/main.js (hunk 2 above)
Comment (indirap) [reply server/main.js:10]: Yeah, it makes eslint ignore just this one line. There's also a separate command for ignoring everything in between some lines

Comment (vatthikorn) [reply server/main.js:10]: That's neat

File: imports/api/groceries.js (hunk 1 above)
Comment (vatthikorn) [reply imports/api/groceries.js:31]: Ah right, I didn't know that also applies here

**Pull request merged**